Documentation/ApiOverview/FlexForms/Index.rst

```diff
@@ -475,6 +475,25 @@ The key `flexform` is followed by the field which holds the FlexForm data
    * :ref:`TypoScript: flexform <t3tsref:data-type-gettext-flexform>`
 
 
+.. index:: pair: FlexForms; Typoscript
```

Comment:
Maybe "Default value" is better than TypoScript? And, it is not TypoScript but page TSconfig ;-)